src/views/Package/components/PackageDocs/Body.tsx

```diff
@@ -72,13 +73,21 @@ const components = ChakraUIRenderer({
 const rehypePlugins = [rehypeRaw];
 
 export const Body: FunctionComponent<{ children: string }> = ({ children }) => {
+  const q = useQueryParams();
+  const format = q.get("format") ?? "pretty";
   const body = useMemo(() => {
-    return (
-      <ReactMarkdown components={components} rehypePlugins={rehypePlugins}>
-        {children}
-      </ReactMarkdown>
-    );
-  }, [children]);
+    if (format === "raw") {
+      return <div>{children}</div>;
+    }
```

Comment:
imo we can probably cut this render format since it was really just for debugging / identifying perf issues